src/globals/scss/core/_prose-scope.scss

```diff
@@ -44,5 +44,9 @@
     ul {
       @extend .govuk-list--bullet;
     }
+
+    a {
```

Comment:
More dragons - if a user wants to use an a tag to define an anchor within a markdown document, e.g.

```
# <a name="section-1">Section 1</a>
```

Then any styling we apply to the .govuk-link class that is not inside a pseudo-selector will be applied to the anchor, despite it not actually being a link.

Bootstrap tackle this by [_undoing_ their link styles using `a:not([href]):not([tabindex])`](https://github.com/twbs/bootstrap/blob/v4-dev/scss/_reboot.scss#L210-L228) which we could add here, but that means the prose scope would need to 'know' what styling the `govuk-link` class is applying in order to undo it.

Alternatively, we could try and ensure we only ever add styling to pseudo selectors within the `govuk-link` class, but that seems hard to document / enforce and also means if anyone wants to override that styling they'll need to apply it to e.g. `:link, :visited` within their own component.

Again, bootstrap has discussed this [at some length](https://github.com/twbs/bootstrap/issues/19402) so worth reading that to try and learn from what they've tried…

Thoughts on a postcard…

Reply:
This could be better explored when we have good variant coverage of the thorny parts of the prose components?

Reply:
Created a card to track this [here](https://trello.com/c/E9qnX9Sf/603-work-out-how-to-handle-non-linking-anchors-within-the-prose-scope).